Update connect-external-services.md (#992)
astronomer/docs: astro/connect-external-services.md

Before you can run pipelines on Astro with real data, you first need to make your data services accessible to your data plane and the Deployments running within it. This guide explains how to securely connect Astro to external data services using the following methods:

- Public endpoints

> - Virtual Private Cloud (VPC)/Virtual Network (VNet) peering 

Reviewer: ```suggestion
- Virtual Private Cloud (VPC)
- Virtual Network (VNet) peering 
```

Author: We don't have a dedicated section on VNet peering, its lumped in with VPC peering.  I'm ok with either way, just seemed off because every other item in this list was its own section in the doc.

To retrieve the IP addresses of an Astro cluster, contact [Astronomer support](https://support.astronomer.io) and request them. Then, allowlist these IP addresses in your data service provider. If you have more than one cluster, you will need to allowlist each cluster individually.

> ## VPC/VNet peering

Reviewer: ```suggestion
## VPC and VNet peering
```

> Each cluster on Astro runs in a dedicated Virtual Private Network (VPC/VNet). To set up private connectivity between an Astro VPC/VNet and another VPC/VNet, you can set up a VPC peering connection. VPC peering ensures private and secure connectivity, reduced network transit costs, and simplified network layouts.

Reviewer: ```suggestion
Each cluster on Astro runs in a dedicated VPC or VNet. To set up private connectivity between an Astro VPC or VNet and another VPC or VNet, you can set up a VPC peering connection. VPC peering ensures private and secure connectivity, reduced network transit costs, and simplified network layouts.
```

To create a VPC peering connection between an Astro VPC and a target VPC, reach out to [Astronomer support](https://support.astronomer.io) and provide the following information:

- Astro cluster ID and name

> - AWS Account ID or Google Cloud project ID or Azure TenantID and Subscription ID of the target VPC/VNet

Reviewer: ```suggestion
- AWS Account ID or Google Cloud project ID or Azure TenantID and Subscription ID of the target VPC or VNet
```

> - Region of the target VPC/VNet (_AWS and Azure only_)

Reviewer: ```suggestion
- Region of the target VPC or VNet (_AWS and Azure only_)
```

> - VPC/VNet ID of the target VPC/VNet

Reviewer: ```suggestion
- VPC or VNet ID of the target VPC or VNet
```

> - CIDR of the target VPC/VNet

Reviewer: ```suggestion
- CIDR of the target VPC or VNet
```

> From there, Astronomer initiates a peering request. To connect successfully, this peering request must be accepted by the owner of the target VPC/VNet in your organization.

Reviewer: ```suggestion
From there, Astronomer initiates a peering request. To connect successfully, this peering request must be accepted by the owner of the target VPC or VNet in your organization.
```

> Once the VPC/VNet peering connection is established, the owner of the target VPC will continue to work with our team to update the routing tables of both VPCs to direct traffic to each other.

Reviewer: ```suggestion
When the VPC or VNet peering connection is established, the owner of the target VPC will continue to work with our team to update the routing tables of both VPCs to direct traffic to each other.
```
